libs/helm/calendar/src/lib/hlm-calendar.ts

```diff
@@ -167,6 +223,16 @@ export class HlmCalendar<T> {
 			),
 	);
 
+	/** Get the heading for the current month */
+	protected readonly _month = computed(() =>
+		this._i18n.config().formatMonth(this._dateAdapter.getMonth(this._calendar().focusedDate())),
+	);
+
+	/** Get the heading for the current year */
+	protected readonly _year = computed(() =>
+		this._i18n.config().formatYear(this._dateAdapter.getYear(this._calendar().focusedDate())),
+	);
+
```

Comment:
Could you move that into _heading computed based on the caption layout input? We would than only generate one heading instead of all three. Do you thing that would be better together or separate?